lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/TypeProcessor.java

```diff
@@ -136,21 +124,29 @@ protected void storeMetadata(Writer writer, Map<String, TypeMetadataEntry> pastE
           .values()
           .forEach(
               entry ->
-                entry.stdlibName().ifPresent(n ->
-                        out.println(
-                        "    public static final String "
-                                + entry.ensoName().toUpperCase()
-                                + " = \""
-                                + n
-                                + "\";")
-                )
+                entry.stdlibName().ifPresent(n -> generateEntry(entry.ensoName().toUpperCase(), n, out))
           );
 
       out.println();
       out.println("}");
     }
   }
 
+  public void generateEntry(String name, String value, PrintWriter out) {
```

Comment:
What is a _"copy based programming"_ error fixed by introducing `generateEntry` method and using it consistently?